app/src/main/java/com/amaze/filemanager/filesystem/HybridFile.java

```diff
@@ -83,7 +82,12 @@
 import net.schmizz.sshj.sftp.SFTPClient;
 import net.schmizz.sshj.sftp.SFTPException;
 
-/** Hybrid file for handeling all types of files */
+/**
+ * Hybrid file for handeling all types of files
+ *
+ * <p>This is deprecated, please use AmazeFile instead
```

Comment:
Whole class is not deprecated yet. 
Also AmazeFile has only the core parts + smb support. Is that the idea here? What's the idea? You're going to migrate all the filesystems helper methods to it?

Reply:
Yes. Now helpers should be moved to the new system.